src/main/java/com/puppycrawl/tools/checkstyle/checks/design/MutableExceptionCheck.java

```diff
@@ -66,6 +66,136 @@
  * <pre>
  * &lt;module name=&quot;MutableException&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ *      class FirstClass extends Exception {
+ *          private int errorCode; // OK Does not extend any class
+ *
+ *          public FirstClass() {
+ *              errorCode = 1;
+ *          }
+ *      }
+ *
+ *      class MyException extends Exception {
+ *          private int errorCode; // violation , The field 'errorCode' must be declared final
+ *
+ *          public MyException() {
+ *              errorCode = 2;
+ *          }
+ *      }
+ *
+ *      class DemoThrowable extends Throwable {
+ *          final int errorCode; // OK
+ *          String message; // violation, The field 'message' must be declared final
+ *
+ *          public DemoThrowable(int errorCode, String message) {
+ *              this.errorCode = errorCode;
+ *              this.message = message;
+ *          }
+ *      }
+ *
+ *      class CustomException extends Throwable { } // OK
+ *
+ *      class ViolationException extends java.lang.Exception {
```

Comment:
`BadException`

Reply:
I think you can remove
```
class CustomException extends Throwable { } // OK
```
CustomException is edge case and rare in real life.

Reply:
Done

Reply:
Not done, use name I suggested to be more compact

Reply:
Done